ui/src/components/flows/FlowCreate.vue

```diff
@@ -9,6 +9,7 @@
             :total="total"
             :guided-properties="guidedProperties"
             :flow-error="flowError"
+            :flow-deprecations="flowDeprecations"
```

Comment:
Maybe call it warning so we can add something else than deprecation later if we need (like security warnings for ex)